content/ja/docs/concepts/workloads/controllers/job.md

```diff
@@ -0,0 +1,662 @@
+---
+title: Jobs
+content_type: concept
+feature:
+  title: バッチ実行
+  description: >
+    サービスだけでなく、KubernetesはバッチとCIワークロードの管理機能も提供し、必要に応じて障害が発生したコンテナを置き換えることもできます。
+weight: 50
+---
+
+<!-- overview -->
+
+Jobは一つ以上のPodを作成し、指定された数のPodが正常に終了するまで、Podの実行を再試行し続けます。Podが正常に終了すると、Jobは成功したPodの数を追跡します。指定された完成数に達すると、そのタスク(つまりJob)は完了したとみなされます。Jobを削除すると、作成されたPodも一緒に削除されます。Jobを一時停止すると、再開されるまで、稼働しているPodは全部削除されます。
+
+単純なケースを言うと、確実に一つのPodが正常に完了するまで実行されるよう、一つのJobオブジェクトを作成します。
+一つ目のPodに障害が発生したり、(例えばノードのハードウェア障害またノードの再起動が原因で)削除されたりすると、Jobオブジェクトは新しいPodを作成してくれます。
+
+Jobで複数のPodを並列で実行することもできます。。
+
+スケジュールに沿ってJob(単一のタスクか複数タスク並列のいずれか)を実行したい場合は [CronJob](/ja/docs/concepts/workloads/controllers/cron-jobs/)を参照してください。
+
+<!-- body -->
+
+## 実行例  {#running-an-example-job}
+
+下記にJobの定義例を記載しています。πの2000桁まで計算して出力するJobで、完了するまで約10秒かかります。
+
+{{< codenew file="controllers/job.yaml" >}}
+
+このコマンドで実行できます:
+
+```shell
+kubectl apply -f https://kubernetes.io/examples/controllers/job.yaml
+```
+
+実行結果はこのようになります:
+
+```
+job.batch/pi created
+```
+
+`kubectl`でJobの状態を確認できます:
+
+```shell
+kubectl describe jobs/pi
+```
+
+実行結果はこのようになります:
+
```

Comment:
Original file has shortcode section at here.
Please check [original file](https://raw.githubusercontent.com/kubernetes/website/main/content/en/docs/concepts/workloads/controllers/job.md), it has following parts:
```
{{< tabs name="Check status of Job" >}}
{{< tab name="kubectl describe job pi" codelang="bash" >}}
```

Reply:
Looks like the original file has been changed. Thank you so much for letting me know!